wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -627,6 +627,43 @@ abstract class Wallet
     } yield tx
   }
 
+  /** @inheritdoc */
+  override def bumpFeeCPFP(
+      txId: DoubleSha256DigestBE,
+      feeRate: FeeUnit): Future[Transaction] = {
+    for {
+      txDbOpt <- transactionDAO.findByTxId(txId)
+      tx <- txDbOpt match {
+        case Some(db) => Future.successful(db.transaction)
+        case None =>
+          Future.failed(
+            new RuntimeException(s"Unable to find transaction ${txId.hex}"))
+      }
+
+      spendingInfos <- spendingInfoDAO.findTx(tx)
+      _ = require(spendingInfos.nonEmpty,
+                  s"Transaction ${txId.hex} must have an output we own")
+
+      changeSpendingInfos = spendingInfos.flatMap { db =>
+        if (db.privKeyPath.chain.chainType == HDChainType.Change) {
```

Comment:
Interesting, do we have no other way to detect if something is a change output ? 

In the future it would probably be nice to decouple detecting change and BIP32. Definitely shouldn't be done in this PR though.

Reply:
ACK